README

```diff
@@ -1,13 +1,13 @@
 =============================================================
 === CrossrefReferenceLinking plugin
-=== Version: 1.0.2
+=== Version: 1.0.3
```

Comment:
We need to standardize the versioning :)
Theoretically, the `stable-3_3_0` branch, which is `1.0.2`, can still receive additions/unexpected breaking changes, then we would risk having collisions.

Reply:
I think we can then use 1.0.2.1, 1.0.2.2 etc. for stable-3_3_0, or?
I see, for example, coins (https://github.com/pkp/coins/releases) and customHeader (https://github.com/pkp/customHeader/releases) work similar... 
But, maybe to double check with @asmecher, at least till we do not decide on something to follow for all...

Reply:
We aren't especially careful/consistent about this. In general we are moving closer to semantic versioning (https://semver.org/).